go/test/endtoend/onlineddl/revertible/onlineddl_revertible_test.go

```diff
@@ -47,7 +47,7 @@ var (
 	tableName             = `onlineddl_test`
 	createTableWrapper    = `CREATE TABLE onlineddl_test(%s)`
 	dropTableStatement    = `
-		DROP TABLE IF EXISTS onlineddl_test
+		DROP TABLE onlineddl_test
```

Comment:
Was this change deliberate to ensure the test fails if the table failed to be created?

Reply:
Sorry for the confusion, but while working on this PR I also fixed an issue with `DROP TABLE` in `-declarative` mode. We shouldn't support it, just like we do not allow `CREATE TABLE IF EXISTS` in `-declarative` mode. This is a nuance; anyway the `endtoend` test used a `IF EXISTS` clause, and shoulnd't have.
I should have made this a separate PR.